.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -0,0 +1,69 @@
+            * Enclave CC: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
+            * Kata Containers:
+              * Note that for Kata Containers, we're looking for the newTag, below the quay.io/confidential-containers/container-engine-for-cc-payload image
+              * s390x: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomizat
+              * x86_64: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+- [ ] - 8. Ensure the Operator is using the latest CI builds and that the Operator tests are passsing
+        * Enclave CC:
+          * SIM: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/sim/kustomization.yaml
+          * HW: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
+          * Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and enclave-cc-{SIM,HW}-latest tags
+        * Kata Containers:
+          * s390x: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomization.yaml
+          * x86_64: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+          * Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and kata-containers-latest tag
+- [ ] - 9. Cut an ocicrypt-rs v<TARGET_RELEASE> release, if changes happened in the project
+- [ ] - 10. Cut an attestation-agent v<TARGET_RELEASE>, if changes happened in the project
+- [ ] - 11. Cut an image-rs v<TARGET_RELEASE> release, using the latest release of:
+        * ocicrypt-rs (redo step 1, but now using v<TARGET_RELEASE>)
+        * attestation-agent (redo step 2, but now using v<TARGET_RELEASE>)
+- [ ] - 12. Cut a td-shim v<TARGET_RELEASE> release, if changes happened in the project
+- [ ] - 13. Update Enclave CC to use the released version of image-rs
+        * redo step 3, but now using v<TARGET_RELEASE>
+- [ ] - 14. Update the operator to use the images generated from the latest commit of both Kata Containers and Enclave CC
+        * redo step 8, but now targetting the latest payload image generated for Kata Containers and Enclave CC
+- [ ] - 15. Make sure all the operator tests are passing
+- [ ] - 16. Cut an Enclave CC release
+- [ ] - 17. Add a new Kata Containers tag
+- [ ] - 18. Update the operator to use the release tags coming from Enclave CC and Kata Containers
+        * redo step 8, but now targetting thje latest release of the payload image generated for Kata Containers eand Enclave CC
+- [ ] - 19. Update the Operator version
+        * https://github.com/confidential-containers/operator/blob/main/config/release/kustomization.yaml#L7
+- [ ] - 20. Cut an operator release
```

Comment:
This isn't something I've been involved in, so might be incorrect, but I think Jens normally updates Operatorhub, so maybe we should add a step to ask him to do that?

Reply:
Added!